lib/sqlite.core.js

```diff
@@ -50,10 +50,14 @@ newSQLError = function(error, code) {
   return sqlError;
 };
 
-nextTick = window.setImmediate || function(fun) {
-      window.setTimeout(fun, 0);
-    };
-
+let nextTick = setImmediate ||  function(fun) {
+  setTimeout(fun, 0);
```

Comment:
where is setTimeout defined?

Reply:
setTimeout is defined in global nodejs, so run in jest test.